indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorSpec.java

```diff
@@ -151,6 +154,21 @@ public DruidMonitorSchedulerConfig getMonitorSchedulerConfig()
   @Override
   public abstract Supervisor createSupervisor();
 
+  /**
+   * need to notice that autoScaler would be null which means autoscale is dissable.
+   * @param supervisor
+   * @return autoScaler, disable autoscale will return dummyAutoScaler and enable autoscale wiil return defaultAutoScaler by default.
+   */
```

Comment:
```suggestion
  /**
   * An autoScaler instance will be returned depending on the autoScalerConfig. In case autoScalerConfig is null or autoScaler is disabled then NoopAutoScaler will be returned.
   * @param supervisor
   * @return autoScaler
   */
```

Reply:
Done. Thanks for your review!